config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1600,9 +1599,9 @@ presubmits:
         env:
           - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
             value: "1"
-
   - name: pull-kubernetes-node-swap-fedora
-    cluster: k8s-infra-prow-build
+# We need an ignition file for swap and k8s-infra-build
+#    cluster: k8s-infra-prow-build
```

Comment:
Do we want to leave those comments here or just provide the ignition within the PR?

Reply:
Hence the hold. We could do them in two but I think it maybe be worth doing them as a separate PR?

WDYT?

I took the ones I could fix and opened a different PR that was merged a few days ago.  These jobs require a new ignition file.

Reply:
Yes, let's do them in separate PR's and use the tracking list to keep the overview: https://github.com/kubernetes/test-infra/issues/30888#issuecomment-1748312747

Reply:
alright, I'm going to just close this one then.